client/src/components/CommitList.jsx

```diff
@@ -10,6 +10,9 @@ const CommitList = props => {
   //  TODO: need to update while using DB
   const { messages } = sprints.allSprints[sprint];
   const milestoneCommits = messages.map(message => message.message);
+  const numberOfMilestoneCommits = commits.filter(commit =>
```

Comment:
What's this about?  It seems non-intuitive to me and I'd love to gain insight into what this intends.

Reply:
I get it now.  Can you rename this variable to something more representative of what it holds?